src/main/java/by/yahor_kulesh/controllers/TicketController.java

```diff
@@ -0,0 +1,30 @@
+package by.yahor_kulesh.controllers;
+
+import by.yahor_kulesh.entity.TicketEntity;
+import by.yahor_kulesh.services.TicketService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+public class TicketController {
+
+    private final TicketService ticketService;
+
+    public TicketController(TicketService ticketService) {
+        this.ticketService = ticketService;
+    }
+
+    @PostMapping("/get")
+    public TicketEntity getTicket(@RequestParam(required = false) UUID id) {
+        return ticketService.getOrCreateTicket(id);
+    }
```

Comment:
even when you create some resource (like Ticket) and use Http method `POST`  you do not need to use verb `get` in url.
better:
1. add to the class `@RequestMapping("/tickets")`
2. to your post method just post: `@PostMapping` withour any additional url